.scripts/fix-i18n.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * This script will:
+ *
+ * - remove any duplicated i18n key on the same file;
+ * - re-order all keys based on source i18n file (en.i18n.json)
+ * - remove all keys not present in source i18n file
```

Comment:
I have seen cases where a "translation" was not maintained in English for single words, where the key is fine as fallback (like `_ "ok"`).
In those cases, I had to add a translation which did not exist in `EN`. => Until there's a verification that all keys contain a translation in `EN`, I'd not remove the translations

Reply:
replying to myself: *if* someone figures out a translation was missing in EN (but the key was used as display text), he/she/it should add the translation to the EN file anyway. I've done the same earlier on as well and now it seems as if there were no translations missing in EN. => Imho, this is fine.

Reply:
yes.. also, if some translation is missing from `EN` nobody will be able to translate it from lingohub, and on the next export from lingohub it would be removed from any other language as well.. so this just keeps lingohub's behavior